web/html/src/build/webpack.config.js

```diff
@@ -9,6 +9,7 @@ const {getWebpackConfig} = require("./webpack.config.vendors");
 module.exports = [
   getWebpackConfig({generateLicenses: true}),
   {
+    // entry: ["@babel/polyfill", pages],
```

Comment:
Is there a reason to keep this comment here?

Reply:
uh I still need to make some tests and some research because of this, to be sure i don't break the application for older browsers. I need to validate if the polyfill prepend we had here: https://build.suse.de/package/view_file/Devel:Galaxy:Manager:Head/susemanager-frontend-libs/gulpfile.js?expand=1 is still needed